cluster/juju/layers/kubernetes-master/README.md

```diff
@@ -40,9 +40,40 @@ firewall rules may not be able to contact the Charm Store. In these network
 restricted  environments the resource can be uploaded to the model by the Juju
 operator.
 
+#### Snap Refresh
+
+Resources used by the kubernetes-master charm are snap packages. By default,
```

Comment:
Unless manually attaching resources, ... ?

Reply:
@hyperbolic2346 good call -- i'll update the readmes to note the `snapd_refresh` only affects snaps from the store.